Sprint-1/implement/sum.js

```diff
@@ -1,4 +1,12 @@
 function sum(elements) {
+  let result = 0;
+
+  for (let i = 0; i < elements.length; i++) {
+    if (typeof elements[i] === "number") {
+      result += elements[i];
+    }
+  }
+  return result;
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function returns the value you expected?

```
sum([Infinity, 1]);
sum([Infinity, Infinity]);
sum([NaN, 1]);
sum([Infinity, -Infinity]);
```